src/ansys_sphinx_theme/__init__.py

```diff
@@ -433,6 +434,28 @@ def add_cheat_sheet(
         context["sidebars"] = sidebar
 
 
+def add_search_option(
+    app: Sphinx, pagename: str, templatename: str, context: Dict[str, Any], doctree: nodes.document
+) -> None:
+    """Control the type of searching used.
+
+    Parameters
+    ----------
+    app : ~sphinx.application.Sphinx
+        Application instance for rendering the documentation.
+    pagename : str
+        Name of the current page.
+    templatename : str
+        Name of the template being used.
+    context : dict
+        Context dictionary for the page.
+    doctree : ~docutils.nodes.document
+        The doctree.
+    """
+    use_pyansys_search = app.config.html_theme_options.get("use_ansys_search", False)
+    context["use_ansys_search"] = use_pyansys_search
```

Comment:
Fixing the naming here:

```suggestion
    use_ansys_search = app.config.html_theme_options.get("use_ansys_search", False)
    context["use_ansys_search"] = use_ansys_search
```